bit_stream_filter_test.go

```diff
@@ -0,0 +1,21 @@
+package astiav
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestBitStreamFilter(t *testing.T) {
+	bitStreamFilterNames := []string{"h264_mp4toannexb", "null", "setts"}
```

Comment:
I don't think we need to test 3 filters here, could you only test the `null` filter?
Testing for an invalid filter name is a good idea however 👍